aix/SPECS/4.2.0/wazuh-agent-4.2.0-aix.spec

```diff
@@ -101,11 +101,23 @@ if [ $1 = 1 ]; then
 fi
 
 if [ $1 = 2 ]; then
-  touch %{_localstatedir}/tmp/wazuh.restart
-  /etc/rc.d/init.d/wazuh-agent stop > /dev/null 2>&1 || :
+  if %{_localstatedir}/bin/wazuh-control status 2>/dev/null | grep "is running" > /dev/null 2>&1; then
```

Comment:
What happens if `wazuh-control` doesn't exists?

Reply:
That issue was fixed in this PR #643